stripe/src/main/java/com/stripe/android/view/StripeEditText.kt

```diff
@@ -218,6 +220,22 @@ open class StripeEditText @JvmOverloads constructor(
         fun displayErrorMessage(message: String?)
     }
 
+    override fun onSaveInstanceState(): Parcelable {
+        return StripeEditTextState(
+            errorMessage,
+            shouldShowError,
+            super.onSaveInstanceState()
```

Comment:
Can we call `super.onSaveInstanceState()` first? Feels more idiomatic, even though I'm not sure it actually makes a difference.